tools/scripts/publish.mjs

```diff
@@ -49,17 +54,10 @@ const copyReadmeCommand = `cp ${rootLib}/README.md ${outputPath}`
 console.log(chalk.bold.greenBright(copyReadmeCommand))
 process.chdir(outputPath)
 
-// Updating the version in "package.json" before publishing
-try {
-  const json = JSON.parse(readFileSync(`package.json`).toString())
-  json.version = version
-  writeFileSync(`package.json`, JSON.stringify(json, null, 2))
-} catch (e) {
-  console.error(chalk.bold.red(`Error reading package.json file from library build output.`))
-}
+
 
 // Execute "npm publish" to publish
-const publishCommand = `npm publish --access public --tag ${tag === 'undefined' ? 'next' : tag}`
+const publishCommand = `npm publish --access public --tag ${tag === 'undefined' ? 'next' : tag} ${otp? `--otp ${otp}`:''}`
```

Comment:
Optional `opt` param, as I needed to publish it locally under my account.
Shouldn't be needed when moving this over to CI 🤞